tomviz/python/BinTiltSeriesByTwo.py

```diff
@@ -4,13 +4,15 @@ def transform_scalars(dataset):
     from tomviz import utils
     import scipy.ndimage
     import numpy as np
+    import warnings
 
     array = utils.get_array(dataset)
 
     zoom = (0.5, 0.5, 1)
     result_shape = utils.zoom_shape(array, zoom)
     result = np.empty(result_shape, array.dtype, order='F')
     # Downsample the dataset x2 using order 1 spline (linear)
+    warnings.simplefilter("ignore")  #suppress warnings
```

Comment:
As far as I know this ignores all warnings, which may not be good. What about something like (I haven't tested it):
`warnings.filterwarnings('ignore', '.*output shape of zoom.*')`
To more specifically ignore the warning we know is not relevant here.

Reply:
That is a good point, it would be better not to filter all possible warnings. I've tested your recommendation and it works as well.